the-9-exercises/src/components/AnimalList.jsx

```diff
@@ -0,0 +1,11 @@
+export default function AnimalList({ id, image, name }) {
+  return (
+    <div
+      className="py-1 flex border items-center gap-7 w-1/3 justify-center"
+      key={id}
+    >
+      <img src={image} alt={`${name} image`} width="70px" />
```

Comment:
[the word 'image' is not recommended to be in alt attribute, ](https://www.freecodecamp.org/news/what-is-alt-text-image-alt-text-html-example/#:~:text=Don't%20include%20the%20words,picture%20when%20writing%20alt%20texts.)